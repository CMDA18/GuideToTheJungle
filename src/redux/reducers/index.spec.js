import Reducers from './index'

describe('Reducers', () => {
  it('Provide a collection of reducer functions', () => {
    expect(Reducers).toMatchSnapshot()
  })
})
