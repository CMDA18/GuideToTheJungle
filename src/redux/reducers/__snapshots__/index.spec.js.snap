// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reducers Provide a collection of reducer functions 1`] = `
Object {
  "settings": [Function],
}
`;
